lib/rust/data-structures/src/unrolled_slot_map.rs

```diff
@@ -1,5 +1,11 @@
 //! A slot map based on [`UnrolledLinkedList`].
 
+// Clippy thinks that `Item` is unused in [`Slot`] definition, but it is not correct.
+// It also does not seem to be possible to disable this lint for the particular struct only.
+
+// === Non-Standard Linter Configuration ===
+#![allow(clippy::extra_unused_type_parameters)]
```

Comment:
That is ridiculous. Seems to be a bug in clippy, but I'm not sure.

Reply:
Seems to be an issues with the code generated by the bytemuck::Zeroable.

Reply:
It was fixed by clippy team 🎉 https://github.com/rust-lang/rust-clippy/pull/11022